_layouts/base.html.haml

```diff
@@ -30,6 +30,17 @@
     %link{:rel => "shortcut icon", :href => relative("/favicon.ico")}
 
   %body(data-spy="scroll" data-target=".bs-docs-sidebar")
+<!--Place GTM Code just below the opening body tag -->
```

Comment:
Removing all html comments to save bandwidth (= improve page loading speed) as this layout applies to all pages.
Added haml comment (which isn't generated into the html page) so it's clear it's for google tag manager.